backend/app/Notifications/ResetPasswordNotification.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Notifications;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+
+class ResetPasswordNotification extends Notification
+{
+    use Queueable;
+
+    /**
+     * The password reset URL.
+     *
+     * @var string
+     */
+    protected $url;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @param  string  $url
+     * @return void
+     */
+    public function __construct(string $url)
+    {
+        $this->url = $url;
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param  mixed  $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        return ['mail'];
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     *
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        return (new MailMessage)
+                    ->line('Forgot Password?')
+                    ->action('Click to reset', $this->url)
```

Comment:
```suggestion
->line("Copy reset token $this->token")
```

notify user to copy reset token